synapse/lib/layer.py

```diff
@@ -1101,27 +1117,29 @@ class Layer(s_nexus.Pusher):
     def __repr__(self):
         return f'Layer ({self.__class__.__name__}): {self.iden}'
 
-    async def __anit__(self, layrinfo, dirn, nexsroot=None, allow_upstream=True, mapasync=True, maxreplaylog=10000):
+    async def __anit__(self, core, layrinfo):
+
+        self.core = core
+        self.loop = asyncio.get_running_loop()
```

Comment:
self.loop is already set on Base objects during Base.__anit__ which is called during Pusher.__anit__.